.circleci/config.yml

```diff
@@ -293,6 +293,19 @@ jobs:
       - run: ./.circleci/brew-deploy.sh
       - slack-notify-on-failure
 
+  trigger-chocolatey-deploy:
+    docker:
+      - image: cimg/base:edge
+    steps:
+      - run:
+          name: Trigger Chocolatey repository build
+          command: |
+            curl -X POST https://circleci.com/api/v2/project/gh/CircleCI-Public/chocolatey-circleci-cli/pipeline \
+              -H 'Content-Type: application/json' \
```

Comment:
```suggestion
              --fail --silent \
              -H 'Content-Type: application/json' \
```

Reply:
This will cause the step to fail if the server returns a non-2xx response. And `--silent` makes the output neater in non-interactive settings.